Sources/XCResultKit/XCResultFile.swift

```diff
@@ -118,6 +118,67 @@ public class XCResultFile {
         }
     }
     
+    /// The export type can be file or directory
+    public enum ExportType: String {
+        case file
+        case directory
+    }
+    
+    
+    /// Export attachment file or folder
+    ///
+    ///     If the folder where the file will be exported doesn't exist, the command will fail
+    ///     with this example: (error: unexpected error at top-level: noEntry)
+    ///
+    /// - Parameters:
+    ///   - id: The id of xcresult object
+    ///   - outputPath: The export output path location
+    ///   - type: The export type
+    ///
+    /// - Note: The outputPath must contained the path of folder and the name of the final exported file with an extension
+    ///
+    public func exportAttachment(id: String, outputPath: String, type: ExportType) {
```

Comment:
Perhaps exportPayload() should be reimplemented as a wrapper to just call this method?

Reply:
fixed